src/index.js

```diff
@@ -1,14 +1,19 @@
-import { BrowserRouter } from 'react-router-dom';
 import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
 import ReactDOM from 'react-dom/client';
+import Store from './redux/configureStore';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Please, add your bootstrap.min.css file into src/styles directory instead of node_modules. The node_modules is ignored while pushing data to GitHub so the CSS would not be accessible.